src/components/NewCommentForm.tsx

```diff
@@ -59,13 +59,14 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
 
     setSubmitting(true);
 
-    // it is very easy to forget about `await` keyword
-    await onSubmit({ name, email, body });
+    try {
+      await onSubmit({ name, email, body });
```

Comment:
Consider adding error handling logic inside the `catch` block to provide feedback to the user in case of submission failure.